web/styles/app_variables.css

```diff
@@ -41,6 +41,19 @@
     /* Keep content from colliding with the search box. */
     --navbar-content-righthand-offset: 5px;
 
+    /*
+    Width of navbar left column
+    40px (Sidebar toggle icon) + 80px (wide logo).
```

Comment:
organization logo can be much wider than that (200px to be exact). Also, it can vary in dark vs light theme.